src/JWTAuthMiddleware.ts

```diff
@@ -59,6 +59,7 @@ export default class JWTAuthMiddleware {
     getToken: (req: any) => Promise<string>;
     beforeUserCreate:(newUser: User, jwtPayload: any) => Promise<any>;
     emailIdentifier: string = "email";
+    internalIdentifier: string = "internalId";
```

Comment:
Should we rename this to `idIdentifier`? 
It's more general than `internalIdentifier `